lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -586,7 +586,22 @@ module.exports = class MethodMappings {
         async clearElementValue(webElementOrId) {
           const element = this.getWebElement(webElementOrId);
           await element.clear();
-
+        
+          // Check if the element still has some value
+          const value = await element.getAttribute('value');
+          
+          // If the element still has value, send BACK_SPACE keys until cleared
+          while (value && value.length > 0) {
+            await element.sendKeys(Key.BACK_SPACE);
+            // Check the value again
+            const updatedValue = await element.getAttribute('value');
+            
+            // Break the loop if the value is cleared
+            if (!updatedValue || updatedValue.length === 0) {
+              break;
+            }
+          }
```

Comment:
We can't be sending keys in a `while` loop :) That would lead to a lot of HTTP calls for the purpose of just clearing an input field. We can send all the keys in a single call as well.